.ember-cli

```diff
@@ -0,0 +1,9 @@
+{
+  /**
+    Ember CLI sends analytics information by default. The data is completely
+    anonymous, but there are times when you might want to disable this behavior.
+
+    Setting `disableAnalytics` to true will prevent any data from being sent.
+  */
+  "disableAnalytics": false
+}
```

Comment:
Don't we generally use Pods (incl. prefix) for our new Projects? Or is this different for addons?

Reply:
Last i checked addon pod structure is not really supported yet but i will look this up.

Reply:
Ok. If addons are different they are different :-) No worries!